### PR TITLE
Add API documentation for release 3.7.7

### PR DESCRIPTION
**Pull request by cinlloc**

### CODE REVIEW

File: apis/en/source/pages/dto-objects/ScanFilesToSynchronize.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.. Copyright 2017 FUJITSU LIMITED
+
+.. _scanfilestosynchronize-object:
+
+scanFilesToSynchronize
+======================
+
+Provides a list of full paths of scanned files to be synchronized retrieved from the :ref:`scansync-object` object.
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/ScanFilesToSynchronize.rst]: delete "synchronized" or "retrieved "

File: apis/en/source/pages/machinescan/overview.rst [hunk 2]
```diff
@@ -15,7 +15,11 @@ Machine Scan API Resources
 	* :ref:`scanPackageFile-get`: Retrieves the list of files from a native package found from a scan
 	* :ref:`scanPackage-getAll`: Retrieves all the native package information from a scan report
 	* :ref:`scanPartition-upload`: Uploads an entire file partition
+	* :ref:`scanSync-create`: Creates a scanSync object
+	* :ref:`scanSync-get`: Gets the meta-data of the scanSync object
```
Comment (emuus) [inline apis/en/source/pages/machinescan/overview.rst]: Returns instead of get

File: apis/en/source/pages/machinescan/overview.rst [hunk 3]
```diff
@@ -15,7 +15,11 @@ Machine Scan API Resources
...
+	* :ref:`scanSync-getFiles`: Gets all full paths of files to be synchronized from a source to a target instance
```
Comment (emuus) [inline apis/en/source/pages/machinescan/overview.rst]: Returns

File: apis/en/source/pages/machinescan/scanSync_getFiles.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+.. function:: GET /users/{uid}/scannedinstances/{siid}/scans/{sid}/sync/{ssid}/files
+
+.. sidebar:: Summary
+
+	* Method: ``GET``
+	* Response Code: ``200 / 304``
+	* Response Formats: ``text/plain`` ``application/xml`` ``application/json``
+	* Since: ``UForge 3.7.7``
+
+Gets all full paths of files to be synchronized from a source to a target instance.
```
Comment (emuus) [inline apis/en/source/pages/machinescan/scanSync_getFiles.rst]: Returns

File: apis/en/source/pages/usercloudaccount/machineImagePublished_getAccount.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+.. function:: GET /users/{uid}/appliances/{aid}/images/{itid}/pimages/{pitid}/accounts/{caid}
+
+.. sidebar:: Summary
+
+	* Method: ``GET``
+	* Response Code: ``200 / 304``
+	* Response Formats: ``application/xml`` ``application/json``
+	* Since: ``UForge 3.7.7``
+
+Retrieve the information of a cloud account.
```
Comment (emuus) [inline apis/en/source/pages/usercloudaccount/machineImagePublished_getAccount.rst]: Returns

File: apis/en/source/pages/usercloudaccount/machineImagePublished_getAccountResources.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+.. function:: GET /users/{uid}/appliances/{aid}/images/{itid}/pimages/{pitid}/accounts/{caid}/resources
+
+.. sidebar:: Summary
+
+	* Method: ``GET``
+	* Response Code: ``200 / 304``
+	* Response Formats: ``application/xml`` ``application/json``
+	* Since: ``UForge 3.7.7``
+
+Retrieves any dynamic resource information from the target cloud environment.
```
Comment (emuus) [inline apis/en/source/pages/usercloudaccount/machineImagePublished_getAccountResources.rst]: Returns

File: apis/en/source/pages/usercloudaccount/scanPublishedImage_getAccount.rst [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+.. function:: GET /users/{uid}/scannedinstances/{siid}/scans/{sid}/images/{itid}/pimages/{pitid}/accounts/{caid}
+
+.. sidebar:: Summary
+
+	* Method: ``GET``
+	* Response Code: ``200 / 304``
+	* Response Formats: ``application/xml`` ``application/json``
+	* Since: ``UForge 3.7.7``
+
+Retrieve the information of a cloud account.
```
Comment (emuus) [inline apis/en/source/pages/usercloudaccount/scanPublishedImage_getAccount.rst]: Returns

File: apis/en/source/pages/usercloudaccount/scanPublishedImage_getAccountResources.rst [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+.. function:: GET /users/{uid}/scannedinstances/{siid}/scans/{sid}/images/{itid}/pimages/{pitid}/accounts/{caid}/resources
+
+.. sidebar:: Summary
+
+	* Method: ``GET``
+	* Response Code: ``200 / 304``
+	* Response Formats: ``application/xml`` ``application/json``
+	* Since: ``UForge 3.7.7``
+
+Retrieves any dynamic resource information from the target cloud environment.
```
Comment (emuus) [inline apis/en/source/pages/usercloudaccount/scanPublishedImage_getAccountResources.rst]: Returns

File: apis/en/source/pages/dto-objects/ApplianceSummary.rst [hunk 9]
```diff
@@ -12,9 +12,9 @@ Attributes
 
 The list of attributes for ``applianceSummary`` are:
 
-	* ``name`` (string): the name of the appliance template
 	* ``version`` (string): the version of the appliance template
-	* ``uri`` (anyURI): the uri of the appliance template
+	* ``name`` (string): the public tag name
```
Comment (jbremond) [inline apis/en/source/pages/dto-objects/ApplianceSummary.rst]: I don't understand why we speak about tag here (abd not use the appliance word)

Comment (cinlloc) [reply apis/en/source/pages/dto-objects/ApplianceSummary.rst]: It's inherited from _SourceSummary_.

**Pull request merged**